packages/babel-plugin/src/__tests__/index.test.tsx

```diff
@@ -129,6 +129,37 @@ describe('babel plugin', () => {
     `);
   });
 
+  it('should preserve comments at the top of the processed file before inserting runtime imports', () => {
+    const output = transformSync(
+      `
```

Comment:
/nit ` on the previous line

Reply:
- [ ]

Reply:
It's prettier placing the ` on a new line